src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(firstNumber) {
```

Comment:
you can write without 'function' 
I mean `return (firstNumber)`

Reply:
Thanks, fixed